controllers/admin/AdminProductsController.php

```diff
@@ -1440,24 +1438,27 @@ public function ajaxPreProcess()
     public function ajaxProcessUpdateProductImageShopAsso()
     {
         $id_product = Tools::getValue('id_product');
-        if (($id_image = Tools::getValue('id_image')) && ($id_shop = (int) Tools::getValue('id_shop'))) {
+        $id_image = Tools::getValue('id_image');
```

Comment:
id_image is casted below. Shouldn't we cast it here ?

Reply:
Fixed